File: Chapter_3/project_3_3.pl
use strict;
use warnings;

printf 'Please enter an ISBN: ';
chomp( my $isbn = <ARGV> );

print "$isbn\n";
my @values = split '-', $isbn;

printf "GS1 prefix:        \t%d\n", $values[0];

printf "Group identifier:  \t%d\n:", $values[1];
printf "Publisher code:    \t%d\n",  $values[2];
printf "Item Number:       \t%d\n",  $values[3];
printf "Check Digit:       \t%d\n",  $values[4];

# return 1;

File: Chapter_3/project_3_3.py
isbn = input("Please enter the ISBN: ")

values = isbn.split('-')

print(f"GS1 prefix:        \t{values[0]}")
print(f"Group identifier:  \t{values[1]}")
print(f"Publisher code:    \t{values[2]}")
print(f"Item Number:       \t{values[3]}")
print(f"Check Digit:       \t{values[4]}")
